apps/frontend/src/libs/enums/notification-message.enum.ts

```diff
@@ -2,6 +2,7 @@ const NotificationMessage = {
 	PROJECT_CREATE_SUCCESS: "Project was successfully created",
 	PROJECT_UPDATE_SUCCESS: "Project was successfully updated.",
 	SUCCESS_PROFILE_UPDATE: "Successfully updated profile information.",
+	SUCCESS_USER_DELETE: "User deleted successfully.",
```

Comment:
I think we should agree how to name this messages, because they are written in different patterns here.